sklearn/utils/fixes.py

```diff
@@ -193,3 +193,54 @@ def __getstate__(self):
                                  self._fill_value)
 else:
     from numpy.ma import MaskedArray    # noqa
+
+
+if np_version < (1, 9, 0):
+    # Allow unique to return counts
+    # https://github.com/numpy/numpy/commit/da3c6a28f651877dfb4cfdbbbe76dab985669252
```

Comment:
The following link might be more relevant:

`https://github.com/numpy/numpy/commit/09fb4205a1d56090e13257a181f23514684f532b`

Reply:
True, thanks!